### PR TITLE
Tweak improved Cut Inner First tooltips

### PR DESCRIPTION
**Pull request by Sophist-UK**

### CODE REVIEW

File: meerk40t/core/planner.py [hunk 1]
```diff
@@ -234,14 +234,11 @@ def shutdown_program():
...
-                    + "using this option can significantly INCREASE the optimisation time. "
-                    + "This option should therefore only be selected if you are cutting the material "
-                    + "i.e. burning right through the material to the other side. "
-                    + "The time taken for Cut Inner First to run can be minimised by: \n"
-                    + "* Putting the inner paths into a separate earlier operation(s) and not using Cut Inner First \n"
-                    + "* Not using Merge Operations"
-                    + "* Using Merge Passes"
+                    "If you find that using this option significantly increases the optimisation time, "
+                    + "alternatives are: \n"
+                    + "* Deselectinbg Cut Inner First if you are not cutting fully through your material \n"
```
Comment (tatarize) [inline meerk40t/core/planner.py:239]: Deselecting has one fewer b.

Comment (tatarize) [reply meerk40t/core/planner.py:239]: Unless that's the UK-English spelling...

**Review by tatarize — APPROVED** [review]
tooltips are harmless to non-translators.

Though admittedly maybe for these very long things we might want to translate `CUT_INNER_FIRST_TOOLTIP` into English of this very long stuff if editing is a thing.

**Comment by Sophist-UK** [comment]
> Though admittedly maybe for these very long things we might want to translate CUT_INNER_FIRST_TOOLTIP into English of this very long stuff if editing is a thing.

The reason to break long strings into paragraphs for translating is to reduce the degree of retranslation if you change a small part of it.

**Review by tatarize — APPROVED** [review]
Tooltips = harmless

**Pull request merged**